plugins/readme/index.js

```diff
@@ -0,0 +1,61 @@
+import chalk from 'chalk';
+import hirestime from 'hirestime';
+import mkdirp from 'mkdirp';
```

Comment:
do we really really _need_ these dependencies? 😅

Reply:
_cmmmmoooon_

Reply:
Its just bloat 😕  if we add dependencies for every little thing, repo is gonna end up like storybook 

whats wrong with `fs.mkdirSync(outdir, { recursive: true });`?

Do we _need_ to log it in _yellow_? Cmooooooon

Reply:
*sigh* aright I boring'd it up for you 😉